### PR TITLE
[ZRH-2018] Add exoscale sponsor

### PR DESCRIPTION
**Pull request by Roderick-Jonsson**

### CODE REVIEW

**Comment by bridgetkromhout** [comment]
If you want a sponsor to have a tracking link specific to you, don't put the tracking link in the sponsor file. You can override a sponsor link in your event's yaml. Here's an example: https://github.com/devopsdays/devopsdays-web/blob/master/data/events/2017-zurich.yml#L161-L163

File: data/sponsors/exoscale.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+name: "Exoscale"
+url: "https://www.exoscale.com/?utm_source=devopsdayszurich2018URLutm_medium=emailURLutm_campaign=event"
```
Comment (phrawzty) [inline data/sponsors/exoscale.yml]: Remove the tracking part - just link to https://www.exoscale.ch here.

File: data/events/2018-zurich.yml [hunk 2]
```diff
@@ -144,6 +144,8 @@ sponsors:
     level: gold
   - id:  redgate
     level: gold
+  - id: exoscale
+    level: bronze
```
Comment (phrawzty) [inline data/events/2018-zurich.yml]: Add a `url` item with the full tracking link here.

**Comment by phrawzty** [comment]
Hello @Roderick-Jonsson,
I've amplified @bridgetkromhout's instructions. The idea here is to have a _generic_ sponsor profile that can be re-used by any event, and then to override sponsor details (such as the URL) on a per-event basis as necessary.

**Comment by bridgetkromhout** [comment]
If you need assistance, @matiasfrndz knows how this works - here is an example of him doing it last year: https://github.com/devopsdays/devopsdays-web/pull/2076

**Comment by Roderick-Jonsson** [comment]
@bridgetkromhout @phrawzty Thank you for your helpful guidance, should be ok now :)

**Pull request merged**